tests/FakeItEasy.Specs/RefReturnSpecs.cs

```diff
@@ -0,0 +1,53 @@
+namespace FakeItEasy.Specs
+{
+    using System;
+    using FakeItEasy.Tests.TestHelpers;
+    using FluentAssertions;
+    using Xbehave;
+    using Xunit;
+
+    /// <summary>
+    /// Characterization tests, representing the current capabilities of the code, not the desired state.

```

Comment:
I don't think it makes sense to add characterization tests for ref return. It makes sense for `in` params, because it's a scenario that should work, but currently doesn't. But faking ref return methods is not something that could ever work, because of fundamental limitations of the platform.

Reply:
I'll drop.